### ISSUE TITLE
Add an option to set testURL manually in Jest config

### ISSUE THREAD

**Issue by jonohayon** [wix/yoshi#1499]
# 🙋 Feature Request

### 😯 Current Behavior

I tried to set the `testURL` parameter in the `jest` field in our project's package.json and the url didn't change.

### 🤔 Expected Behavior

The URL should change.

### 💁 Possible Solution 

Add a yoshi config option to set that URL, either as a string or even better as a result of a function.

### 🔦 Context 

I'm in the process of moving one of `chat`'s front end apps from mocha to Jest. Our tests use a given URL on a specific port as the base for all of the http mocking, which we used to set ourselves to our custom jsdom instance (since mocha doesn't provide one for us) and there were no problems with it.
Currently, though, there's no official way to change the global `jsdom` instance's URL (the instance that Jest uses) - see facebook/jest#5124. There are a bunch of solutions to this, the easiest to implement IMO would just be to provide a `testURL` option in the yoshi config, or a better one would be to use the custom [jest-environment-jsdom-global](https://www.npmjs.com/package/jest-environment-jsdom-global) which exposes the Jest jsdom instance globally for all of the tests.

**Comment by jonohayon** [comment]
I am, of course, willing to implement this feature (at least the easier solution as I'm pretty much behind schedule on this whole jestification process)

**Comment by ranyitz** [comment]
@jonohayon I don't understand the use-case (And therefore I'm not sure that the best way to solve it would be to configure the `testURL`).

> Our tests use a given URL on a specific port as the base for all of the http mocking

Can you share a link to this, it would be interesting to understand.

In any case, @yanivefraim did some work to make those kinds of overrides possible. https://github.com/wix/yoshi/pull/1467 If we decide it's the best way we can make it configurable.

**Comment by jonohayon** [comment]
I looked on Yaniv's PR (and in the docs before writing this issue), and it seems as if the only property that's allowed to be overridden is the `globals` property.

>  Can you share a link to this? it would be interesting to understand.
I confused the port with a query parameter (both are important but for the sake of the tests the port can be switched out to port 80) that's required for getting an instance id. We currently use the aforementioned solutions - more specifically, the history API - to change the url to include that query param but IMO it's a bit hacky and it would be better if we could just specify the browser test URL.

**Comment by ranyitz** [comment]
> If we decide it's the best way we can make it configurable.

Yes, it is pretty hacky. We'll accept a PR that adds `testUrl` to the supported override keys.

Do you want to contribute it?

**Comment by jonohayon** [comment]
Yeah I'll work on it

**Comment by ranyitz** [comment]
@jonohayon Great! Let us know if you have any question

**Comment by jonohayon** [comment]
Created a PR with the changes for implementing this - #1506
